src/MetaDataItem.py

```diff
@@ -1,2 +1,25 @@
+import json
+
 class MetaDataItem:
-    pass
+    def __init__(self, id, title, url, collision_type, description, location):
+        self.id = id
+        self.title = title
+        self.url = url
+        self.collision_type = collision_type
+        self.description = description
+        self.location = location
+        self.obj = {
+            'title': self.title,
+            'url': self.url,
+            'collision_type': self.collision_type,
+            'description': self.description,
+            'location': self.location
+        }
```

Comment:
Why do we need to keep extra pointers like that to objects we already own?
```suggestion
```

Reply:
I don't understand your question

Reply:
Check my latest commit. I only create the JSON dict when the functions are called.